src/ApplicationLogs/ApplicationLogs.csproj

```diff
@@ -6,16 +6,16 @@
     <RootNamespace>Neo.Plugins</RootNamespace>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\LevelDBStore\LevelDBStore.csproj" />
+    <ProjectReference Include="..\RpcServer\RpcServer.csproj" />
+  </ItemGroup>
```

Comment:
Why modify this file?

Reply:
I re-referenced these two projects manually, so the sequence changed. It was an unnecessary change. I'll fix this.